src/Service.Tests/SqlTests/GraphQLPaginationTests/MsSqlGraphQLPaginationTests.cs

```diff
@@ -25,55 +25,92 @@ public static async Task SetupAsync(TestContext context)
 
         /// <inheritdoc />
         [DataTestMethod]
-        [DataRow("typeid", 1, 4, "", "",
+        [DataRow("typeid", 1, 4, "", "", false,
```

Comment:
How did you determine whether to assign true/false to `isLastPage` given just these param values?

Reply:
By looking at the records we populate the DB with, if the request will land on the last record, we set last page to true, otherwise there is another record available and we set it to false.